README.md

```diff
@@ -1,17 +1,22 @@
-# Documentation
+# Epiphany Platform
+[![GitHub release](https://img.shields.io/github/v/release/epiphany-platform/epiphany.svg)](https://github.com/epiphany-platform/epiphany/releases)
+[![Github license](https://img.shields.io/github/license/epiphany-platform/epiphany)](https://github.com/epiphany-platform/epiphany/releases)
 
 ## Overview
 
 Epiphany at its core is a full automation of Kubernetes and Docker plus additional builtin services/components like:
 
 - Kafka or RabbitMQ for high speed messaging/events
 - Prometheus and Alertmanager for monitoring with Graphana for visualization
-- Elasticsearch and Kibana for centralized logging
+- Elasticsearch and Kibana for centralized logging (OpenDistro)
 - HAProxy for loadbalancing
-- Postgress for storage
-- KeyCloak for authentication
+- Postgress and Elasticsearch for date storage
+- Vault (MVP) and KeyCloak for authentication
+- Helm as package manager for Kubernetes
 
-Epiphany can run on as few as one node (laptop, desktop, server) but the real value comes from running 3 or more nodes for scale and HA. Nodes can be added or removed at will depending on data in the manifest. Everything is data driven so simply changing the manifest data and running the automation will modify the environment.
+Following target platforms are available: AWS, Azure and on-prem installation.
+
+Epiphany can run on as few as one node (laptop, desktop, server) but the real value comes from running 3 or more nodes for scale and HA. Nodes can be added or removed at will depending on data in the initial manifest. Everything is data driven so simply changing the manifest data and running the automation will modify the environment.
```

Comment:
@toszo Not sure about meaning of "at will" in that case..

Instead of:
> Nodes can be added or removed at will depending on data in the initial manifest.

Shouldn't it be: 
> Nodes can be added or removed depending on data in the initial manifest.